learn/advanced/known_limitations.md

```diff
@@ -100,3 +100,9 @@ user = 1 OR user = 2 […] OR user = 1500 OR user = 1501 […] OR user = 2000 OR
 **Limitation:** By default, Meilisearch returns up to 1000 documents per search.
 
 **Explanation:** Meilisearch limits the maximum amount of returned search results to protect your database from malicious scraping. You may change this by using the `maxTotalHits` property of the [pagination index settings](/reference/api/settings.md#pagination-object). `maxTotalHits` only applies to the [search route](/reference/api/search.md) and has no effect on the [get documents endpoint](/reference/api/documents.md#get-documents).
+
+## Large datasets and internal errors
+
+**Limitation:** Meilisearch might throw an internal error when indexing large datasets.
```

Comment:
We should quantify "large" as we do in the other Known limitations

Reply:
The problem is that we don't have an unambiguous answer as in other items. This limit can be 2GB CSV/3.5GB JSON in a system, but might be higher in some machines and lower in others.

My idea was to be sort of vague at first and then give a generic explanation in the following paragraph.